Formula/xorriso.rb

```diff
@@ -1,9 +1,9 @@
 class Xorriso < Formula
   desc "ISO9660+RR manipulation tool"
   homepage "https://www.gnu.org/software/xorriso/"
-  url "http://ftpmirror.gnu.org/xorriso/xorriso-1.4.0.tar.gz"
-  mirror "https://ftp.gnu.org/gnu/xorriso/xorriso-1.4.0.tar.gz"
-  sha256 "0bd1e085015b28c24f57697d6def2fe84517967dc417554c0c3ccf1685ed0e56"
+  url "https://www.gnu.org/software/xorriso/xorriso-1.4.2.tar.gz"
```

Comment:
I wonder whether `ftpmirror` was intentionally favored over the primary site. @DomT4 probably knows …